dependencies.yaml

```diff
@@ -1207,6 +1208,7 @@ dependencies:
       - output_types: conda
         packages:
           - cudf-polars==25.10.*,>=0.0.0a0
+          - pytest-httpserver
```

Comment:
This doesn't belong in this `depends_on_cudf_polars` list.

* these `depends_on_*` lists are only intended to serve one dependency
* placing this here makes `pytest-httpserver` a runtime dependency of `cudf-polars` (which I believe you didn't intend)

Please move this to `cudf-polars` testing dependencies instead:

https://github.com/rapidsai/cudf/blob/8921686cef85b2e5d9040937c661a369405d1f63/dependencies.yaml#L889